src/script/common/c_content.h

```diff
@@ -162,6 +164,10 @@ bool               push_json_value           (lua_State *L,
 void               read_json_value           (lua_State *L, Json::Value &root,
                                               int index, u8 recursion = 0);
 
+void               push_pointed_thing        (lua_State *L, const PointedThing &pointed);
+
+void               push_objectRef(lua_State *L, const u16 id);
```

Comment:
even if this looks (IMO) horrible as it was done before, align it the same way as the other functions, please.